zest/releaser/prerelease.py

```diff
@@ -127,6 +132,24 @@ def _grab_history(self):
         # TODO: add line number where an extra changelog entry can be
         # inserted.
 
+        # Look for 'Nothing changed yet' under the latest header.  Not
+        # nice if this text ends up in the changelog.  Did nothing happen?
+        start = headings[0]['line']
```

Comment:
If someone has a few extra lines at the top of the history file, ten lines may not be enough. If you increase it to twenty you may get false positives for earlier releases where you forgot this. Not really bad of course.
I could change it, but in this case I thought: we already have the headers so we know exactly where to look, so let's use it. Is that okay or should I still change it?